runtime/onert/core/src/ir/train/TrainableGraph.cc

```diff
@@ -127,6 +127,13 @@ const ITrainableOperation &TrainableGraph::operation(OperationIndex index) const
   return dynamic_cast<const ITrainableOperation &>(_graph.operations().at(index));
 }
 
+void TrainableGraph::enableBackward(const OperationIndex &index)
+{
+  auto op = dynamic_cast<ir::train::ITrainableOperation *>(&_graph.operations().at(index));
```

Comment:
ditto
```suggestion
  auto op = nnfw::misc::polymorphic_downcast<ir::train::ITrainableOperation *>(&_graph.operations().at(index));
```

Reply:
Unfortunately, virtual inherited objects cannot be static_casted. It's also mentioned in https://github.com/Samsung/ONE/blob/5b7c3108314cc5c3c1ecbc17f9b9023826505b36/runtime/onert/core/src/ir/train/TrainableGraph.cc#L126 but here I need non-const operation.
With this change I see the error:
```
cannot convert from pointer to base class ‘onert::ir::IOperation’ to pointer to derived class ‘onert::ir::train::ITrainableOperation’ because the base is virtual
```

Reply:
I see. right. Thanks for your information.